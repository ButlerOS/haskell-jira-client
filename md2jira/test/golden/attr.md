# epic

## story {#P-4 points=42 updated=2024-09-07}

- [ ] task {.end}
note
