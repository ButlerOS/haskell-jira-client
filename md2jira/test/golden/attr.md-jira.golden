EpicTitle: epic
EpicBody:


StoryTitle: story
StoryBody:
* ☐ task
note

